utils/viewmodel/src/main/kotlin/com/bumble/appyx/utils/viewmodel/ActivityIntegrationPointWithViewModel.kt

```diff
@@ -0,0 +1,15 @@
+package com.bumble.appyx.utils.viewmodel
+
+import android.os.Bundle
+import androidx.activity.ComponentActivity
+import com.bumble.appyx.navigation.integration.ActivityIntegrationPoint
+
+class ActivityIntegrationPointWithViewModel(
```

Comment:
Similarly to `NodeActivity` we'll have to provide `ViewModelNodeActivity` which will use `ActivityIntegrationPointWithViewModel`